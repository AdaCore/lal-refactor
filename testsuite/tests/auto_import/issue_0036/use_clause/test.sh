auto_import -P test.gpr --source src/main.adb --line 2 --column 12
