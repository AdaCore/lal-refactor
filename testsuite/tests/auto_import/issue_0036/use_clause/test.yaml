description:
  This test checks that we correcly add missing import
  for use-clauses that reference toplevel packages.

driver: shell_script
