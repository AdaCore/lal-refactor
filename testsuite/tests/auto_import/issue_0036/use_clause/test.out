Available Imports:
Import: Parent; Qualifier: 

Edits:
main.adb
1:1-1:1 with Parent;


